.github/workflows/release.yaml

```diff
@@ -0,0 +1,83 @@
+name: Release Flow
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  publish-packages:
+    name: Push Packages
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        python-version: ["3.10"]
+    steps:
+      - uses: actions/checkout@master
+      - uses: actions/setup-python@v3
+        with:
+          python-version: ${{ matrix.python-versions }}
+      - uses: addnab/docker-run-action@v3
+        with:
+            image: valory/open-autonomy-user:latest
+            options: -v ${{ github.workspace }}:/work
+            run: |
+              echo "Pushing Packages"
+              cd /work
+              export AUTHOR=$(grep 'service' packages/packages.json | awk -F/ '{print $2}' | head -1)
+              autonomy init --reset --author $AUTHOR --ipfs --remote
+              autonomy push-all
+  publish-images:
+    name: Publish Docker Images
+    runs-on: ${{ matrix.os }}
+    needs:
+      - "publish-packages"
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        python-version: ["3.10"]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up tag and vars
+        uses: addnab/docker-run-action@v3
+        with:
+            image: valory/open-autonomy-user:latest
+            options: -v ${{ github.workspace }}:/work
+            run: |
+              echo "Setting Tag Images"
+              cd /work
+              echo "VERSION=0.8.0" > env.sh
+              echo AUTHOR=$(grep 'service' packages/packages.json | awk -F/ '{print $2}' | head -1) >> env.sh
+              echo SERVICE=$(grep 'service' packages/packages.json | awk -F/ '{print $3}' | head -1) >> env.sh
+              echo DEFAULT_IMAGE_TAG=$(cat packages/packages.json | grep agent | awk -F: '{print $2}' | tr -d '", ') >> env.sh
+              cat env.sh
+
+      - uses: addnab/docker-run-action@v3
+        name: Build Images
+        with:
+            image: valory/open-autonomy-user:latest
+            options: -v ${{ github.workspace }}:/work
+            shell: bash
+            run: |
+              echo "Building Docker Images"
+              cd /work
+              source env.sh || exit 1
+              echo "Building images for $AUTHOR for service $SERVICE"
+              autonomy fetch $AUTHOR/$SERVICE --service --local || exit 1
+              cd $SERVICE || exit 1
+              autonomy build-image || exit 1
+              autonomy build-image --version $VERSION || exit 1
+
+
+      - name: Docker login
+        env:
+          DOCKER_USER: ${{secrets.DOCKER_USER}}
+          DOCKER_PASSWORD: ${{secrets.DOCKER_PASSWORD}}
+        run: |
+            echo  $DOCKER_PASSWORD | docker login -u $DOCKER_USER --password-stdin 
```

Comment:
Assume this will work...

Reply:
hard to test without auth

Reply:
Looks similar to an example from the [docs](https://docs.docker.com/engine/reference/commandline/login/#-provide-a-password-using-stdin---password-stdin), so should be good